simulator/host_system.go

```diff
@@ -173,6 +173,20 @@ func CreateStandaloneHost(f *Folder, spec types.HostConnectSpec) (*HostSystem, t
 	return host, nil
 }
 
+func (h *HostSystem) DestroyTask(req *types.Destroy_Task) soap.HasFault {
+	task := CreateTask(h, "destroy", func(t *Task) (types.AnyType, types.BaseMethodFault) {
+		f := Map.getEntityParent(h, "Folder").(*Folder)
```

Comment:
We probably need to check that the host doesn't have any VMs running on it, may need to be in disconnected state, etc.  ok to leave as-is for now though, maybe just add a TODO.

Reply:
@dougm I see. Will make the change.